### DISCUSSION TITLE
you can add other types of strategy optimization

### DISCUSSION THREAD

**Discussion by Brianramirez27** [kernc/backtesting.py#454] [General]
Hello, I would like to know if it is possible to add other types of optimization to strategies other than those that are in the library such as genetic algorithms or other ways to optimize

**Comment by kernc** [comment]
Everything is possible if you can code it. :smirk:

What other optimization types would you opt for?

**Reply by Brianramirez27** [reply]
The truth is I would like to apply an optimization to my strategy with genetic algorithms after performing the proper bactesting, what I am not clear about is whether the library after doing the bacquesting allows adding that type of optimization
